lambdas/delete-organisation/deploy-lambda.sh

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env bash
```

Comment:
All of those lambda specific deploy scripts have been removed. Instead there is single script in the project root. 
It would be better to make those adaptions there

Reply:
how about publishing the base layers at the start of the deploy.sh script
via: https://docs.aws.amazon.com/cli/latest/reference/lambda/publish-layer-version.html

and then updating the layers configuration for every lambda thats getting deplyoed via: https://docs.aws.amazon.com/cli/latest/reference/lambda/publish-layer-version.html

This would make sure that the deployed lambdas base layers match the commons version.


I know there are probably better solutions making use of cloudformation etc. But for now this would be simple solution that gives as small lambda zips and fixed base layer dependencies